src/load_secrets_from_vault/load_secrets_from_vault/load.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+
+import hvac
+
+import load_secrets_from_vault.context
+
+from lib.util import (
+  url_host_port
+)
+from lib.vault import Vault
+
+
+def load_secrets(artsy_project, vault_host, vault_port, secrets_file, kvv2_mount_point):
+  ''' load secrets from  Vault and write them to a file '''
+  logging.info(f'Loading secrets from {vault_host}:{vault_port} under {kvv2_mount_point}')
+
+  path = f'kubernetes/apps/{artsy_project}/'
+
+  vault_client = Vault(
+    url_host_port(vault_host, vault_port),
+    auth_method='kubernetes',
+#    auth_method='iam',
```

Comment:
will this be used in future? 👀

Reply:
yeah, i want to later on revise this to detect where the script is run. if it's run in k8s, use `kubernetes` auth method, if on local, use `iam`. i've been manually switching back and forth by commenting one or the other.